tests/serverpod_test_server/migrations/serverpod_test/20231103144707/definition.sql

```diff
@@ -362,7 +362,7 @@ ALTER TABLE ONLY "object_with_parent"
     ADD CONSTRAINT "object_with_parent_fk_0"
     FOREIGN KEY("other")
     REFERENCES "object_field_scopes"("id")
-    ON DELETE CASCADE
+    ON DELETE NO ACTION
```

Comment:
Question - Should this one really be changed? This was introduced before the relation keyword was introduced right? Shouldn't we preserve the actions for those tests?

Reply:
I think having the new default makes sense unless it would break a test. In this case it does not cause any issues.